mlperf_logging/compliance_checker/mlp_parser/__init__.py

```diff
@@ -4,11 +4,12 @@
 
 
 def parse_file(filename, ruleset='0.6.0'):
-    if ruleset == '0.6.0':
+    ruleversion = ruleset.split("_")[-1]
```

Comment:
I think we may need separate ruleset files for hpc and training.

Reply:
I have fixed this, this was actually not necessary.